src/utils/github.ts

```diff
@@ -0,0 +1,40 @@
+// Utils for the GitHub API
+
+import axios from 'axios';
+import dotenv from 'dotenv';
+dotenv.config();
+
+const API_LINK = "https://api.github.com";
+
+export interface GitHubRepository {
+  name: string,
+  full_name: string,
+  owner: {
+    avatar_url: string,
+  }
+  html_url: string,
+}
+
+export interface GithubRepositoryRelease {
+  tag_name: string,
+}
+
+// Get repository information
+export const getRepositoryInfo = async (owner: string, repo: string): Promise<GitHubRepository> => {
+  let response = axios.get(`${API_LINK}/repos/${owner}/${repo}`, {
+    params: {
+      auth: process.env.GITHUB_ACCESS_TOKEN,
```

Comment:
how safe is this
plz lint
looks good :)

Reply:
This should be safe? Unless there's another better method of doing things.
Linting has been fixed, and I attached screenshots to the PR